src/sc/embedder/tests/keywords.robot

```diff
@@ -56,9 +60,21 @@ Update
     Input Text  css=${description_selector}  ${description}
     Click Element  css=${image_input_selector}
     Select File  image.jpg
+    Wait For Condition  return tinyMCE.activeEditor != null
+    Execute Javascript
+    ...  var editors = tinyMCE.editors;
+    ...  if (editors[0] === undefined) {
```

Comment:
This code is used to access each editor at Plone 4.2
Unfortunately RF javascript expression don't allow to add a comment